modules/storage-persistent-storage-pv.adoc

```diff
@@ -150,7 +150,7 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 //|Ceph RBD  | ✅ | ✅ |  -
 //|CephFS  | ✅ | ✅ |  ✅
 |Cinder  | ✅ | - |  -
-|Fibre Channel  | ✅ | ✅ |  -
```

Comment:
I think it would be beneficial to mention that only [raw block volumes](https://docs.openshift.com/container-platform/4.14/storage/understanding-persistent-storage.html#block-volume-support_understanding-persistent-storage) with `ReadWriteMany`. The filesystem ones may (and will) fail in a horrible way.

Reply:
So I think the better way is adding a note/annotation under the table (Table 2. Supported access modes for persistent volumes), but not “check mark”.

Reply:
@aglitke Please check on the suggestions and let me know what is to be added in the note as per the suggestion.

Reply:
I like the check mark but could we use an asterisk (*) or footnote to clarify that it's only for block mode?

Reply:
okay from me.

Reply:
@aglitke and @duanwei33 the changes are made. Please check and confirm